packages/terra-avatar/src/Avatar.jsx

```diff
@@ -98,3 +98,4 @@ Avatar.propTypes = propTypes;
 Avatar.defaultProps = defaultProps;
 
```

Comment:
Should we be exporting these via the Opts object to remain consistent with other components? See [alert](https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.jsx#L205) for an example.

Reply:
Yeah, we should try and be consistent with our other components that expose variants.

Reply:
Discussed in war room. Plan is to start moving to separate exports for additional data or utils components may need.